### DISCUSSION TITLE
Github Support

### DISCUSSION THREAD

**Discussion by coder0107git** [jgm/djot#312] [General]
I just found [this](https://github.com/github/markup) and thought it might be useful for anyone in the future trying to figure out how to add Djot rendering support to Github.

**Comment by ryanabx** [comment]
Wrote a quick PR, but I don't have much time to help maintain this. Any help would be appreciated! :)

https://github.com/github/markup/pull/1861

**Comment by tolmasky** [comment]
I saw that this pull request was never responded to, did you get any other feedback from the by any chance? Is using ruby a requirement? I saw you pointed out that there was no ruby parser, but couldn't find that listed as a requirement anywhere.

I'd love to see this happen so if you have any insights from giving it a try, I'd love to see if I could perhaps help.
